packages/replayio/src/commands/remove.ts

```diff
@@ -15,6 +15,10 @@ registerCommand("remove")
   .action(remove);
 
 async function remove(shortIds: string[], { all = false }: { all?: boolean }) {
+  if (all) {
+    removeFromDisk();
```

Comment:
nit: it's not particularly clear what this function does when it doesn't receive any argument. I'd suggest splitting its implementation into two functions, one would require the recording id and the other one would be called smth like `removeAllRecordings`